evalml/data_checks/outliers_data_check.py

```diff
@@ -56,11 +53,16 @@ def get_IQR(df, k=2.0):
             upper_bound = q3 + (k * iqr)
             return (lower_bound, upper_bound)
 
-        clf = IsolationForest(random_state=self.random_state)
-        clf.fit(X)
-        scores = pd.Series(clf.decision_function(X))
-        lower_bound, upper_bound = get_IQR(scores, k=2)
-        outliers = (scores < lower_bound) | (scores > upper_bound)
-        outliers_indices = outliers[outliers].index.values.tolist()
-        warning_msg = "Row '{}' is likely to have outlier data"
-        return [DataCheckWarning(warning_msg.format(row_index), self.name) for row_index in outliers_indices]
+        lower_bound, upper_bound = get_IQR(X)
```

Comment:
I know we default `k` to 2 in the signature but couldn't hurt to include `k=2` here.

Actually... I think it'd be cool if this were an init param, but fine to file separately if you don't wanna lump it into this PR